src/System.Diagnostics.Process/tests/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Windows_NT;
       netstandard-Unix;
+      uap-Windows_NT;
+      uapaot-Windows_NT;
```

Comment:
I recall you said without this, uapaot will fail. is there issue tracking the problem caused you to add such configuration? or did you ask Wes about this problem?